### PR TITLE
Add Bug Issue Templates

### LINKED ISSUE #203: Add Bug Template for Filing bugs in issues
- We should add bug/issue templates so that people are forced to describe certain aspects of the bugs

### PR DESCRIPTION
**Pull request by jonathan-innis**
Closes #203

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```
Comment (nitishm) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: We probably do not need these two. Maybe some contextual ones like,
- Create/Update/Delete the ApplicationGroup?

**Review by nitishm — COMMENTED** [review]
Minor nit

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jonathan-innis) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Changed it slightly, tell me what you think

**Review by nitishm — APPROVED** [review]
🚀

**Pull request merged**